### DISCUSSION TITLE
How to locate plots associated with a plotnft?

### DISCUSSION THREAD

**Discussion by Proulx-S** [Chia-Network/chia-blockchain#9026] [Q&A]
Hello,

Situation: I have ~1% of my plots scattered across 7 different plotnfts, and the rest associated to a single plotnft.

Goal: I want to get ride my 1% scattered plots.

Problem: I don't know where the scattered plots are.

Question: How can I locate plots associated with a given plotnft.

Thanks for the help!

**Comment by b-a-s-s** [comment]
I have a similar problem for a different task.

I have many plotnft and need to switch pool for some.

With `chia plotnft show`, I can find the plotnft that needs to switch pool.
With `chia wallet show`, I can find wallet ids required by `chia plotnft join`.
But I can't find the match between the plotnft and the corresponding wallet id...

Any help would be most welcome!
Thanks!

**Reply by BrandtH22** [reply]
Hey @b-a-s-s , when running `chia wallet show` you should see a "Pool Wallet" and that would be the wallet id to enter for the `chia plotnft join` command.

If you do not see this wallet then I recommend performing a wallet resync by:
```
1. Stop chia
2. Rename the wallet db (~\.chia\mainnet\wallet\db\) - the wallet db stores offer files and cat/did names so renaming is for archive purposes in case you need that data
3. Start chia and wait for sync 
4. Try the show wallet command once synced and look for the Pool Wallet
```

Let me know if any questions arise and take note that we can generally provide more timely and thorough support in our discord (https://discord.gg/chia).
